### ISSUE TITLE
Not an issue - Hoping someone can help with PHP storm setup please?

### ISSUE THREAD

**Issue by JackWormUK** [clean-docker/Magento2#92]
Hi Everyone

Not an issue (the setup was impressively smooth). I am wondering if someone can point me to a guide on how to setup PHPStorm with this project?  I have moved over from Vagrant so I am still figuring out docker.

This really is a great and easy to install project and it is a credit to @rafaelstz generosity to share this.  I very much appreciate it!

**Comment by iods** [comment]
Still having issues? Worked a bit with this, let me know if you still need help.

**Comment by JackWormUK** [comment]
@iods Yes I am.  I gave up in the end and went back to trusty Notepad++.  Any help would be appreciated.

**Comment by JackWormUK** [comment]
@iods Are you able to help in any way with this in anyway please?

**Comment by iods** [comment]
@JackWormUK what issues are you experiencing?

**Comment by JackWormUK** [comment]
@iods I am totally new to PHPStorm.  I managed to get it working on a local wamp (linking up the Config) but i have no idea how to link it into docker.  As i don't know what i am doing i don't know which online guides i should be following or if they are relevant to my use case.

I appreciate setting it up will take some time; links to relevant and correct guides would be a huge help if you know of any?  I am also happy to buy a book.

**Comment by iods** [comment]
@JackWormUK shoot me a mail, lets take this out of here. rye@drkstr.dev
